### DISCUSSION TITLE
cpg0020-varchamp

### DISCUSSION THREAD

**Discussion by rsenft1** [broadinstitute/cellpainting-gallery#61] [Dataset descriptions]
Imaging Platform project tag = 2021_09_01_VarChAMP_Pilot
Segmentation/ Feature extraction is being performed by `Cimini lab`
Profile creation is being performed by `Cimini lab`
Data can be public in RODA Immediately - `yes`

Update as generated:  
[Link to profile repo](https://github.com/broadinstitute/2021_09_01_VarChAMP_Vidal_Taipale)
[Link to publication repo]  
[cpg0020-varchamp-pilot]

- [ ] Metadata collection form filled out by collaborator or Imaging Platform member (https://airtable.com/shrVxz9DcoMlDoCBI)
- [ ] Metadata completely filled out in Project Profiler Database (Imaging Platform internal use only)
- [X] Segmentation/Feature extraction complete
- [X] Profiling complete

Transfer to CellPainting Gallery:
- [X] Upload data to RODA (is private by default)
- [ ] Run validation script to ensure completion
- [ ] Update cellpainting-gallery/README.md
- [x] Make RODA entry public

If data is being published, prepare for publication:
- [ ] Run Distributed-BioFormats2Raw to create .ome.zarr files
- [ ] Upload (meta)data to IDR (images remain hosted in cellpainting-gallery).

Once published:
- [ ] Make IDR entry public
- [ ] Update cellpainting-gallery/README.md and open-data-registry/cellpainting-gallery.yml to reflect publication
- [ ] Move this Issue from cellpainting-gallery-private to cellpainting-gallery. This step can be performed at an earlier point if it needs inputs from an external collaborator.

**Comment by shntnu** [comment]
@ErinWeisbart asked:

As per a conversation in check-in last week, I believe that my last task related to this batch is to move it to the cellpainting-gallery. A project for 2021_09_01_VarChAMP_Vidal_Taipale exists in the cellpainting-gallery already (cpg0020-varchamp-pilot) and has the first arrayed batch in it (2022_08_22_Batch_1).

If I'm understanding correctly, this batch (B1A1R1) represents the first production batch, not a pilot batch.
- Should I create a new cpg for production data (starting with this batch)?
- Should I instead rename the gallery identifier to (cpg0020-varchamp) (removing "pilot") and put this batch in the same folder?
- Should I move the other pilot arrayed batch to the gallery as well? (2023_02_23_Batch_3)
- Should I move pooled pilot batches to the gallery as well? (2023_01_11_Batch2 and 2023_06_05_24W_RD3_poscon/ (in progress))

**Comment by shntnu** [comment]
> * Should I instead rename the gallery identifier to (cpg0020-varchamp) (removing "pilot") and put this batch in the same folder?

Please do this; simpler with a one-off overhead of syncing data over across prefixes (just to be sure: instead of renaming, sync it over and then delete the source)

> * Should I move the other pilot arrayed batch to the gallery as well? (2023_02_23_Batch_3)

 know nothing about this pilot specifically, but it makes for all of them to live together, assuming this is indeed VarCHAMP

> * Should I move pooled pilot batches to the gallery as well? (2023_01_11_Batch2 and 2023_06_05_24W_RD3_poscon/ (in progress))

Oh wait – that's unrelated to VarCHAMP, right? (it's pooled rare?)

**Comment by ErinWeisbart** [comment]
To confirm:
- I will sync arrayed pilot batches `2022_08_22_Batch_1`, `2023_02_23_Batch_3` and this first batch of production data `B1A1R1` to a new gallery identifier `cpg0020-varchamp`
- I will then delete the previously created `cpg0020-varchamp-pilot`
- Pooled batches are considered a different project despite having the same `2021_09_01_VarChAMP_Vidal_Taipale` prefix and will not be synced to this prefix

**Comment by GaiaGerbaka** [comment]
As part of the CPG harmonization, I found two incorrect files in the metadata. `2023-05-26_B1A1R1_P2T2.txt`has the incorrect `plate_map_name`, it should be 2023-05-26_B1A1R1_P2T2 but instead it shows 2023-05-26_B1A1R1_P2T3.` 2023-05-29_B1A1R1_P3T2.txt` has the incorrect `plate_map_name`,  it should be 2023-05-29_B1A1R1_P3T2 but instead it shows 2023-05-29_B1A1R1_P3T1. The rest of the information in the file is correct.

**Reply by ErinWeisbart** [reply]
Fixed as of today
